### DISCUSSION TITLE
Add sync to supported features tags for apps

### DISCUSSION THREAD

**Discussion by kevinclevenger** [theDanielJLewis/podcasting2.org#12] [Ideas]
For those that use multiple devices it would be helpful to indicate which apps support subscription and/or state (played, progress, etc) sync. Just Sync would do, or maybe one of SubSync/StateSync/FullSync.

**Comment by theDanielJLewis** [comment]
Thanks! Although that's a thing more for the other site I'm building (slowly): canipodcast.com, which would show the comparison of apps and features like this.
